module/config.mjs

```diff
@@ -2035,6 +2035,112 @@ DND5E.currencies = {
 };
 preLocalize("currencies", { keys: ["label", "abbreviation"] });
 
+/* -------------------------------------------- */
+
+/**
+ * @typedef CraftingConfiguration
+ * @property {CraftingCostsMultiplier} consumable     Discounts for crafting a magical consumable.
+ * @property {Record<string, CraftingCosts>} magic    Magic item crafting costs.
+ * @property {CraftingCostsMultiplier} mundane        Multipliers for crafting mundane items.
+ * @property {CraftingCosts} potionOfHealing          Crafting costs for a potion of healing.
```

Comment:
Such an awkward exception. Not sure if it's worth bothering with really.

Reply:
We could add a configuration object, mapping together cost and uuids, then shove Potion of Healing in there. More work but less awkward and future proofs it in case more of these outliers would appear in future content.

Reply:
Yeah that's probably better. That solves my issue of which UUID is the 'canonical' Potion of Healing. SRD? PHB? What about the SRD 5.2? With a configuration object we would just add all of the UUIDs. We could also consider using the `identifier`. A potion of healing is a potion of healing, regardless of where it came from or who created it.

Reply:
Agree that using `identifier` would be better. Also means the '24 PHB (and whichever other content modules) will not require adding any data.

Reply:
It sounds like a perfect use of `identifier` 👍

Reply:
Alright I have removed the `potionOfHealing` property and added the `exceptions` property, which is an object of `CraftingCosts` keyed by Item identifier. Actually taking these exceptions into account when calculating crafting costs is still a TODO though because I am out of time.